pkg/chart/loader/load.go

```diff
@@ -74,6 +74,7 @@ func LoadFiles(files []*BufferedFile) (*chart.Chart, error) {
 	subcharts := make(map[string][]*BufferedFile)
 
 	for _, f := range files {
+		c.Raw = append(c.Raw, &chart.File{Name: f.Name, Data: f.Data})
```

Comment:
It looks like all of the `Raw` files will be stored on the release object (by default a secret). This is going to cause them to balloon in size and we will run into storage space problems.

What happened to the change that deleted `Raw` from the release object?

Reply:
When I tested this manually, I did not see `Raw` field stored.